revenuecat_examples/MagicWeather/lib/src/views/user.dart

```diff
@@ -36,7 +36,7 @@ class _UserScreenState extends State<UserScreen> {
 
     try {
       if (task == "login") {
-        await Purchases.logIn(newAppUserID);
+        await Purchases.logIn(newAppUserID!);
```

Comment:
Hmm wonder if we should separate the `login` operation to a different method to avoid the optional and avoid this...

Reply:
I split _manageUser into three functions so it's safer